pkgs/development/python-modules/pbr/default.nix

```diff
@@ -11,13 +16,19 @@ buildPythonPackage rec {
 
   propagatedBuildInputs = [ setuptools ];
 
-  # circular dependencies with fixtures
+  # check in passthru.tests.pytest to escape infinite recursion with fixtures
   doCheck = false;
+
+  passthru.tests = {
+    pytest = callPackage ./tests.nix { };
```

Comment:
This package uses `stestr` not `pytest`.

Reply:
That doesn't change anything but you are right.